dbus/mocks/DBusAPI.go

```diff
@@ -17,8 +17,9 @@
 package mocks
 
 import (
-	dbus "github.com/mendersoftware/mender/dbus"
 	mock "github.com/stretchr/testify/mock"
+
+	dbus "github.com/mendersoftware/mender/dbus"
 )
 
 // DBusAPI is an autogenerated mock type for the DBusAPI type
```

Comment:
This one being an auto-generated file, I think we should not change.

Reply:
Good catch! Did not strike me!

Reply:
Should be safe though, considering we will just regenerate it next time, but still, let's keep it pristine - thanks!